__rebuildDatabase.py

```diff
@@ -0,0 +1,50 @@
+import mysql.connector
+import os
+
+try:
+    # Connect to the MySQL database
+    host = os.getenv('DB_HOST', 'localhost')
+    user = os.getenv('DB_USER', 'root')
+    password = os.getenv('DB_PASSWORD', 'root')
+    port = os.getenv('DB_PORT', 8889)
+
+    print("Connecting to MySQL...")
+
+    db_connection = mysql.connector.connect(
+        host=host,
+        user=user,
+        password=password,
+        port=port
```

Comment:
#apparently it should also be here as the last line in this parenthesis

database = database_name